source/Subsystem/EntityComponentSystem.cpp

```diff
@@ -9,6 +11,13 @@ bool EntityComponentSystem::Init()
 
 void EntityComponentSystem::Update(double dt)
```

Comment:
dt is not following the code style guide I believe. Also, it can be made const as it shouldn't be changed.

Reply:
You are correct.